ddtrace/contrib/trace_utils.py

```diff
@@ -302,3 +305,14 @@ def flatten_dict(
         else:
             flat[p] = v
     return flat
+
+
+def parse_version(version):
```

Comment:
I'm using something like this in https://github.com/DataDog/dd-trace-py/pull/2374 so we can not call this file `trace_utils`?
It has nothing to do with tracing.

Reply:
cool! yeah, I wasn't sure if we wanted to move a level higher or not. the only place I saw it really used (or well, the whole `__version.split(".")` code, was in integration.

I'll move!